eq-author/src/App/questionConfirmation/Design/index.js

```diff
@@ -99,6 +99,7 @@ export const UnwrappedQuestionConfirmationRoute = ({
   return (
     <EditorLayout
       title={data?.questionConfirmation?.displayName || ""}
+      pageDescription={data?.questionConfirmation?.pageDescription || ""}
```

Comment:
```suggestion
```
This line can be removed as EditorLayout does not have a pageDescription prop, and so is unused